### PR TITLE
Update README file with vf lag and large MTU

### PR DESCRIPTION
**Pull request by ameerm93**
Add HowTo configure large MTU
Add HowTo configure bonds
Add HowTo pull image from docker hub

Signed-off-by: Ameer Mahagneh <ameerm@nvidia.com>

### CODE REVIEW

**Comment by ameerm93** [comment]
@wmousa @moshe010, please review

File: README.md [hunk 1]
```diff
@@ -6,7 +6,103 @@ Also it implement two ovs modules:
...
+    vim /etc/sysconfig/network-scripts/ifcfg-slave2
+        NAME=<Desired name>
+        DEVICE=<PF2>
+        TYPE=Ethernet
+        BOOTPROTO=none
+        ONBOOT=no
+        MASTER=bond1
+        SLAVE=yes
+        NM_CONTROLLED=no
+    ```
```
Comment (wmousa) [inline README.md:103]: So I think you should add here the ifup for bond interface and also it's better to check that vf-lag configured with
"dmes | grep lag"

File: README.md [hunk 2]
```diff
@@ -18,18 +114,65 @@ $ MLNX_OFED_VERSION=50218 /bin/bash build.sh
 ```
```
Comment (wmousa) [inline README.md:115]: 52220

File: README.md [hunk 3]
```diff
@@ -18,18 +114,65 @@ $ MLNX_OFED_VERSION=50218 /bin/bash build.sh
...
 Now the ovs-docker image created successfully
 
+## Pull docker image from docker hub 
+
+  - Install and start docker:
+    ```
+    yum install docker
```
Comment (wmousa) [inline README.md]: Did you verify this?
I think it should be something like this: 
yum install -y yum-utils device-mapper-persistent-data lvm2 mstflint bind-utils
yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
yum install -y docker-ce docker-ce-cli containerd.io
systemctl enable docker
systemctl start docker
or at least redirect it to docker.com page https://docs.docker.com/engine/install/centos/

File: README.md (hunk 1 above)
Comment (ameerm93) [reply README.md:103]: done

File: README.md (hunk 3 above)
Comment (ameerm93) [reply README.md]: i only use yum install docker
but maybe its better to redirect to the how to

File: README.md (hunk 2 above)
Comment (ameerm93) [reply README.md:115]: done

File: README.md (hunk 3 above)
Comment (ameerm93) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (moshe010) [reply README.md:103]: I confusing now. change to 

Create linux bonding interface
    ```

Bring up bond interface 
  ```    
       ifup bond1
```

File: README.md [hunk 4]
```diff
@@ -18,18 +115,64 @@ $ MLNX_OFED_VERSION=52220 /bin/bash build.sh
 ```
 Now the ovs-docker image created successfully
 
+## Pull docker image from docker hub 
+
+  - Install(https://docs.docker.com/engine/install/centos/) and start docker:
```
Comment (moshe010) [inline README.md]: Change to:
 - `Install and start docker https://docs.docker.com/engine/install/centos/`_
 remove Line 121-123

File: README.md [hunk 5]
```diff
@@ -18,18 +115,64 @@ $ MLNX_OFED_VERSION=52220 /bin/bash build.sh
...
+    docker images
+    ```
+  - clone containerized-ovs-forwarder:
+    ```
+    git clone https://github.com/Mellanox/containerized-ovs-forwarder
+    ```
+  - change directory to containerized-ovs-forwarder:
+    ```
+    cd containerized-ovs-forwarder/
+    export MLNX_OFED_VERSION=<ofed version example: 52220>
```
Comment (moshe010) [inline README.md]: remove L 140 it note needed

File: README.md [hunk 6]
```diff
@@ -18,18 +115,64 @@ $ MLNX_OFED_VERSION=52220 /bin/bash build.sh
...
+    ```
+    docker pull mellanox/ovs-forwarder:52220
+    ```
+  - check image pulled successfully
+    ```
+    docker images
+    ```
+  - clone containerized-ovs-forwarder:
+    ```
+    git clone https://github.com/Mellanox/containerized-ovs-forwarder
```
Comment (moshe010) [inline README.md:132]: we should do 
git checkout origion/v5.2 -b v5.2 (one I create a branch)

File: README.md [hunk 7]
```diff
@@ -41,69 +184,45 @@ $ MLNX_OFED_VERSION=52220 /bin/bash container_create.sh --pci-args 0000:02:00.0
...
-$ docker start ovs-forwarder:$MLNX_OFED_VERSION
-```
-
 ## Enable debugging inside OVS container  
 To enable debug in ovs inside the container, you can run the following command inside the container:  
   ```
   $ ovs-appctl vlog/set dpdk::DBG
   ```  
+## MTU / jumbo frame configuration
+  - Verify kernel on VM is 4.14+
```
Comment (moshe010) [inline README.md:199]: why the kernel on the VM important?

File: README.md [hunk 8]
```diff
@@ -41,69 +184,45 @@ $ MLNX_OFED_VERSION=52220 /bin/bash container_create.sh --pci-args 0000:02:00.0
...
+    ```
+    tcpdump -i ens4f0 -nev icmp &
+    ping 11.100.126.1 -s 9188 -M do -c 1
+    ```
+  - Enable host_mtu in xml add the following values to xml 
+    ```
+    <domain type='kvm' id='21' xmlns:qemu='http://libvirt.org/schemas/domain/qemu/1.0'>
+    <qemu:commandline>
+    <qemu:arg value='-set'/>
+    <qemu:arg value='device.net1.host_mtu=9216'/>
```
Comment (moshe010) [inline README.md:217]: we missing page per vq as well. Don't you want to mention it?

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (ameerm93) [reply README.md:217]: done

File: README.md (hunk 7 above)
Comment (ameerm93) [reply README.md:199]: you can take a look here: https://redmine.mellanox.com/issues/2182810
it explains the issue

File: README.md (hunk 6 above)
Comment (ameerm93) [reply README.md:132]: will be modified after adding the branch

File: README.md (hunk 5 above)
Comment (ameerm93) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (ameerm93) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (ameerm93) [reply README.md:103]: done

**Review by moshe010 — APPROVED** [review]

**Pull request merged**